paper-slider.html

```diff
@@ -113,6 +113,7 @@
       <paper-input id="input" class="slider-input" disabled$="[[disabled]]" on-change="_inputChange"></paper-input>
     </template>
 
+    <input type="hidden" value="[[value]]" name="[[name]]">
```

Comment:
I just saw `IronFormElementBehaviour`. I think that behavior makes sense, but I will argue that elements such as `paper-slider` should work inside a regular form as well.